arelle/CntlrCmdLine.py

```diff
@@ -256,6 +256,8 @@ def parseArgs(args):
                       help=_("Log reference object properties (default)."), default=True)
     parser.add_option("--logNoRefObjectProperties", "--lognorefobjectproperties", action="store_false", dest="logRefObjectProperties",
                       help=_("Do not log reference object properties."))
+    parser.add_option("--logXmlMaxLength", "--logxmlmaxlength", action="store", dest="logXmlMaxLength", type="int",
+                      help=_("Log file XML field max length override."))
```

Comment:
It's not clear to me what this does based only on the name and help info. Perhaps `--logXmlMaxAttributeLength` and `help=_("Truncate XML log file attribute values at length."))`?  Also set this to the default of `128`?

Reply:
> Also set this to the default of 128

The pattern we've established thus far with `RuntimeOptions` is that all arguments are optional, and the default behavior for when the value is absent is defined wherever the argument is used. If we define default values in the parser, any direct use of `RuntimeOptions` would not have access to that. We're creating a potential difference between CLI and API usage.

Reply:
Perhaps just update the help text to make it clear that the unset value is 128 then?